public/New-PSModule.ps1

```diff
@@ -0,0 +1,78 @@
+﻿function New-PSModule {
```

Comment:
### I don't like this.

I think we want the `PowerShellTemplate` to **be** the template. Not _contain_ the template, or have functions for generating a template.  The so-called `CloneModule` seems _too_ empty to me.

Additionally, the attempt to duplicate a subset of the functionality of `dotnet new` is going to drag us down -- preventing us from using any functionality that we haven't duplicated.

I would rather **tell** people to just `git clone https://github.com/questionmark/PowerShellTemplate.git` and regenerate the GUID.

Reply:
When I started into dotnet new, I wasn't sure how to do this. I'm reasonably sure I know what to do now, with any luck. Should be a simple change, though I suppose I need something else to demonstrate .\data inclusion